.gitignore

```diff
@@ -1,3 +1,4 @@
 /target
 **/*.rs.bk
 Cargo.lock
+*~
```

Comment:
Your editor creates backups? Or where does this come from?
(Not that I am complaining about it, I am just curious what creates these files)

Reply:
emacs does this

Reply:
This should be part of [`.git/info/exclude`](https://help.github.com/en/github/using-git/ignoring-files#excluding-local-files-without-creating-a-gitignore-file) instead.